### PR TITLE
Increase web socket timeout to handle azure EOF exceptions in testing pods

### PR DESCRIPTION
**Pull request by roastario**

### CODE REVIEW

**Review by Schife — APPROVED** [review]
Looks good

File: buildSrc/src/main/groovy/net/corda/testing/KubesTest.java [hunk 1]
```diff
@@ -64,6 +64,10 @@
      */
     private static final String REGISTRY_CREDENTIALS_SECRET_NAME = "regcred";
 
+    private static int DEFAULT_K8S_TIMEOUT_VALUE_MILLIES = 60 * 1_000;
```
Comment (barrylapthorn) [inline buildSrc/src/main/groovy/net/corda/testing/KubesTest.java:67]: `MILLIS` <pedantic/>

**Review by barrylapthorn — APPROVED** [review]

File: buildSrc/src/main/groovy/net/corda/testing/KubesTest.java (hunk 1 above)
Comment (roastario) [reply buildSrc/src/main/groovy/net/corda/testing/KubesTest.java:67]: :man_facepalming:

Comment (barrylapthorn) [reply buildSrc/src/main/groovy/net/corda/testing/KubesTest.java:67]: Don't get me started on `asynch`.

**Pull request merged**